### DISCUSSION TITLE
how did you get sound to work in chromium/firefox?

### DISCUSSION THREAD

**Discussion by develperbayman** [Joshua-Riek/ubuntu-rockchip#19] [General]
the preinstalled chromium would only load a green screen in videos with sound so i uninstalled then reinstalled as a snap seems better but no sound thru hdmi only in the browsers but sound works in the system

**Comment by Joshua-Riek** [comment]
The preinstalled chromium and sound should work fine. Which version are you using?

**Reply by develperbayman** [reply]
whichever one from the snap store but i cannot seem to get sound to play in any browser video but no sound vivaldi browser said something about not having h.24 codecs and will not be able to play media?? .....i started a new job so i havent been able to do much

**Reply by develperbayman** [reply]
prob dont matter just seen you updated to 1.6 gonna test it

**Reply by Joshua-Riek** [reply]
I can't support anything from the snap store, so I have been hesitant to include it by default. Since to get browsers working correctly for this device, they need to be compiled from the source and tweaked to support the hardware. It usually takes a long time for snaps or apt packages to be updated with these tweaks.

The preinstalled chromium should be working fine for video playback and sound, if it's acting strange I would do a fresh install. There is not much I can do about any other browsers.
